src/postprocess/apirequests.py

```diff
@@ -1,131 +1,33 @@
-"""
-Enable and add roles to selected users.
-
-Example:
-
-  import dhis2api
-  import process
-  api = dhis2api.Dhis2Api('http://localhost:8080/api',
-                           username='admin', password='district')
-  users = process.select_users(api, usernames=['test.dataentry'])
-  process.add_roles(api, users, ['role1', 'role2'])
-"""
-
-import sys
-import time
-import requests
+import datetime
 import json
+from datetime import time
 
-import dhis2api
-
-
-def postprocess(cfg, entries, import_dir):
-    """Execute actions on the appropriate users as specified in entries.
-
-    The entries structure looks like:
-        [
-            {
-                "selectUsernames": ["test.dataentry"],
-                "selectFromGroups": ["program1", "program2"],
-                "action": "addRoles",
-                "addRoles": ["role1", "role2"]
-            },
-            {
-                "selectFiles": ["users.json"],
-                "action": "import",
-                "importStrategy": "CREATE_AND_UPDATE",
-                "mergeMode": "MERGE",
-                "skipSharing": "false"
-             }
-        ]
-
-    `action` can be "activate", "deleteOthers", "addRoles" or
-    "addRolesFromTemplate", with an additional field for "addRoles" (a list)
-    and "addRolesFromTemplate" (a string) if that's the action.
-    """
-    api = dhis2api.Dhis2Api(cfg["url"], cfg["username"], cfg["password"])
+import requests
 
-    wait_for_server(api)
+from d2apy import dhis2api
```

Comment:
I guess that we are using https://pypi.org/project/d2apy, we may add a dependency note on the readme.

And probably a requirements.txt: https://blog.usejournal.com/why-and-how-to-make-a-requirements-txt-f329c685181e